setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup, find_packages
+import sys
+
+
+if sys.version_info < (3, 7):
+    sys.exit("Sorry, Python < 3.7 is not supported")
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+version_string = "v0.0.1"
+
+setup(
+    name="pyloopkit",
+    version=version_string,
+    author="Tidepool",
+    author_email="ed@tidepool.org",
+    description="Python implementation of the Loop algorithm",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/tidepool-org/PyLoopKit",
+    packages=find_packages(),
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD-2-Clause',
+        'Programming Language :: Python :: 3.7',
+    ],
+    install_requires=[
+          'numpy==1.16.4',
```

Comment:
I would recommend we add the following back
          'pandas==0.24.2',
          'plotly==4.1.0',
          'matplotlib==3.1.1',

Reply:
I removed those as they aren't used in the core lib (as evidenced by tests passing), and were just used by the `example.py` file.  If we do end up vectorizing this, then tensorflow would need to be added back in. But, like any library with possibly many uses, we should minimize dependencies, and keep the set of responsibilities small so that client applications aren't impacted with possibly conflicting dependencies.

I'd like to make a loop report analyzer based on the example.py file, but that can be a separate package.

Reply:
Also been playing with implementing a tool to look at mean forecast error for Loop on a given data set, so we can understand implications of algorithm changes, and build settings optimization tools on it.